### ISSUE TITLE
Address accessibility issues with the dashboard

### ISSUE THREAD

**Issue by ollylucl** [CLOSER-Cohorts/dashboard#51]
**User story**

As a user, I want to be able to view content on the dashboard that is accessible and is WCAG2.2 compliant.

**Task description**

The tabs at the top of the screen be displayed vertically to comply with https://www.w3.org/TR/WCAG22/#reflow.

At present, if we zoom in by 400%, the tab headings look like this:

![image](https://github.com/CLOSER-Cohorts/dashboard/assets/147418528/c325dabd-79d7-48ef-beff-60608c44c17b)

We need to apply CSS styling (e.g. flex-wrap:wrap to the container holding the tab headings so they are stacked on top of each other as a column when we zoom in, and 'width:min-content text-align: justify' to the buttons so the text in the buttons wraps when we zoom in) in order to address this issue.

![image](https://github.com/CLOSER-Cohorts/dashboard/assets/147418528/9b750caa-d3a3-4c9e-bafa-7f3ea52eb7cb)

Also, the links within the tabs can't be tabbed to. i.e. the links to 'Adolescence', 'Childhood', 'Early adulthood', etc. in the screenshot below cannot be accessed using just a keyboard. We need to add the attribute 'tabindex=0' to each of these links in order that they can be tabbed to using a keyboard and the site complies with https://www.w3.org/TR/WCAG22/#keyboard.

![image](https://github.com/CLOSER-Cohorts/dashboard/assets/147418528/74259193-5556-4b62-a643-585a6249ec00)

The WAVE plugin also detected that the language of the dashboard page couldn't be determined, we need to specify that the language of the page is English in order to comply with https://www.w3.org/TR/WCAG22/#language-of-page.

**Acceptance criteria**

All of the above issues are addressed, and the page is WCAG2.2 compliant.
